keyboards/tetris/rules.mk

```diff
@@ -36,26 +36,23 @@ F_USB = $(F_CPU)
 # Interrupt driven control endpoint task(+60)
 OPT_DEFS += -DINTERRUPT_CONTROL_ENDPOINT
 
+
 # Boot Section Size in *bytes*
-#OPT_DEFS += -DBOOTLOADER_SIZE=4096
+OPT_DEFS += -DBOOTLOADER_SIZE=4096
```

Comment:
Please revert the changes to this file. 

Namely, 

`BOOTLOADER = qmk-dfu` sets the bootloader size, so that's not needed. 

And please remove all of the question marks.

And add `LAYOUTS = LAYOUT_ortho_5x12`